managegroup.php

```diff
@@ -239,9 +239,10 @@
 
 	$plugins->run_hooks("managegroup_do_manageusers_start");
 
-	if(is_array($mybb->get_input('removeuser', MyBB::INPUT_ARRAY)))
+	$users = $mybb->get_input('removeuser', MyBB::INPUT_ARRAY);
+	if(is_array($users) && count($users))
```

Comment:
The line above warranties it will be always an array, doesn't it? It might be better and simpler for it to be `!empty($users)`.

Reply:
```suggestion
	if(!empty($users))
```